README.md

```diff
@@ -1,19 +1,353 @@
 
-Annotated JSON.
+AtJSON is a collection of repositories that together make up a fully-realized content format.
 
-## Core module
+| Quick Links |
+|-------------|
+| [How does a document get rendered?](#how-does-a-document-get-rendered) |
+| [How do I use existing documents?](#how-do-i-use-existing-documents) |
+| [How do I create a new document?](#how-do-i-create-a-new-document) |
 
-FIXME intro docs here.
+The breakdown of modules in this repository are:
 
-[AtJSON](packages/core)
+| Modules | Description|
+|---------|------------|
+| [@atjson/core](packages/core) | AtJSON document and annotation code |
+| [@atjson/hir](packages/hir) | HIR (Heirarchical Intermediate Representation) |
+| [@atjson/renderer-hir](packages/renderer-hir) | Abstract base class for text-based output |
+| [@atjson/renderer-plain-text](packages/renderer-plain-text) | Plain text output |
+| [@atjson/renderer-react](packages/renderer-react) | React output |
+| [@atjson/renderer-commonmark](packages/renderer-commonmark) | CommonMark output |
+| [@atjson/contenttype-commonmark](packages/contenttype-commonmark) | Conversion of CommonMark sources to AtJSON |
+| [@atjson/contenttype-html](packages/contenttype-html) | Conversion of HTML sources to AtJSON |
 
-### Internals
 
-HIR (Heirarchical Intermediate Representation): [@atjson/hir](packages/hir)  
-Plain Text Renderer: [@atjson/plain-text-renderer](packages/plain-text-renderer)  
-CommonMark Renderer: [@atjson/commonmark-renderer](packages/commonmark-renderer)  
-React Renderer: [@atjson/react-renderer](packages/react-renderer)  
+#### Why another content format?
+
+Our goals at Condé Nast are to provide best-in-class content to our readers, and best-in-class tools for our editors. To do this, we need a format that can be rich, extensible, and portable. It is important that the content source provide little ambiguity about _how_ a story should be displayed.
+
+For Condé Nast, it is essential for our stories to be a format that doesn't age quickly- we have over a hundred years of articles from publications across the world. It is extremely important for us to be able to put these stories written for digital media in our archive for the future. Many current content formats do not strive for rigorousness in this, instead tying the constraints of the format to the current platform of choice. When constructing this format, we chose to separate the original text from the data associated for this purpose.
+
+#### What does a document look like?
+
+A document has a `content` field and a list of `annotations`, each of which have a spatial offset in the document.
+
+The simplest document is one without annotations:
+
+```js
+{
+  "content": "Hello"
+  "annotations": []
+}
+```
+
+Stylistic annotations are easily layered on a document using a positional annotation:
+
+```js
+{
+  "content": "The best writing anywhere, everywhere.",
+  "annotations": [{
+    type: "italic",
+    start: 4,
+    end: 8
```

Comment:
and `// exclusive` ?

Reply:
We count in-between letters I believe. 🙃  Let me make sure, but I think you're correct

Reply:
I can also make a diagram that could make this easier